### PR TITLE
Search button design & functionality

### LINKED ISSUE #3: Integrate search functionality
> AS A user
> I WANT to see list of songs
> SO THAT I can see the summary of the songs information

When a user searches for a song title, it should present the top 5 most relevant results from a music database based API (e.g. Spotify, Shazam)
The API response should also feed into another API call to search for the associated Wikipedia article and provide a summary of the song, and some other details.

Overall once a user presses SEARCH, they should see:
- Song title
- Artist
- Album
- Wikipedia summary

### PR DESCRIPTION
**Pull request by kevinvongmany**

### CODE REVIEW

File: assets/css/styles.css
Comment (kevinvongmany) [inline assets/css/styles.css:1]: Turn this css into tailwind classes in the HTML, happy for you to leave the colour stylings

**Review by kevinvongmany — COMMENTED** [review]
Please change as per feedback

**Review by Cbeattie97 — APPROVED** [review]
Looking Good

**Review by kevinvongmany — COMMENTED** [review]
Looks great!

File: searchbutton.html
Comment (kevinvongmany) [inline searchbutton.html:1]: Looks great!

**Review by kevinvongmany — COMMENTED** [review]
Looks great!

**Review by leontran44 — APPROVED** [review]

**Pull request merged**